lua/github_access.lua

```diff
@@ -7,25 +7,23 @@ local github_api_enabled            = os.getenv('GITHUB_API_ENABLED')
 local github_api_company            = os.getenv('GITHUB_API_COMPANY')
 local env_github_restriction_upload = os.getenv('GITHUB_USER_ALLOWED_UPLOAD')
 local github_auth_cache_dir         = "/data/nginx/artifacts_github_auth_cache"
-local github_restriction_upload = {}
```

Comment:
use a more generic approach to restrict on upload AND copy (write operations are not protected in a consistent way).